.travis.yml

```diff
@@ -8,16 +8,24 @@ matrix:
       os: osx
       osx_image: xcode8.2
       before_install:
+        - rvm list
+        - rvm install .ruby-version 
```

Comment:
As far as I know the `rvm install` command expects a ruby version not a filename. So, instead of `rvm install .ruby-version` it should be something like `rvm install $(cat .ruby-version)`.